src/styles.css

```diff
@@ -15,6 +15,13 @@
   --color-border-light: oklch(30% 0.01 290);
   --color-accent: oklch(83% 0.05 70);
   --color-accent-light: oklch(93% 0.05 70);
+
```

Comment:
This approach works, but it is too complicated for what we need. You can replace it all with `light-dark()` function, and by doing this little change you will be able to revert all changes in local components.

https://drafts.csswg.org/css-color-5/#light-dark